packages/@ionic/cli/src/commands/deploy/manifest.ts

```diff
@@ -22,19 +31,27 @@ export class DeployManifestCommand extends DeployCoreCommand {
   async getMetadata(): Promise<CommandMetadata> {
     return {
       name: 'manifest',
-      type: 'project',
+      type: 'global',
```

Comment:
I had to switch this command to a type of `global` to avoid throwing an exception if `ionic.config.json` is not present.

Reply:
I would put this as a comment in the code so it is clear why it is like this

Reply:
Done.